actor/src/main/scala/org/apache/pekko/serialization/Serialization.scala

```diff
@@ -457,8 +454,12 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     }
   }
 
+  @nowarn("msg=deprecated")
   private def warnUnexpectedNonAkkaSerializer(clazz: Class[_], ser: Serializer): Boolean = {
```

Comment:
Could we change the function name to use 'NonPekko'?

Reply:
Makes sense